TunableLaser/v4/_base.yaml

```diff
@@ -0,0 +1,12 @@
+type: Main
+host: laser-control.cp.lsst.org
+port: 4001
+wavelength:
+  min: 300
+  max: 1250
+optical_configuration: SCU
+timeout: 1
+temp_ctrl:
+  host: laser-control.cp.lsst.org
+  port: 4002
+    
```

Comment:
You have this:
![image](https://github.com/user-attachments/assets/f4c284c6-3ac0-4347-af6a-216452cba3bb)